projects/core/components/button/button-options.ts

```diff
@@ -1,22 +1,29 @@
-import {InjectionToken} from '@angular/core';
+import {InjectionToken, ValueProvider} from '@angular/core';
 import {TuiAppearance} from '@taiga-ui/core/enums';
 import {TuiSizeXL, TuiSizeXS} from '@taiga-ui/core/types';
 
-export interface ButtonOptions {
+export interface TuiButtonOptions {
```

Comment:
@splincode @vladimirpotekhin that's a breaking change, do you think we should keep the old interface too but deprecate it?

Reply:
plz don't forget the same for input-count, input-password, checkbox etc